.github/workflows/release.sh

```diff
@@ -102,6 +109,11 @@ kerl_configure() {
     KERL_BUILD_DOCS=yes
     export KERL_BUILD_DOCS
 
+    export_kerl_configuration_option "--disable-dynamic-ssl-lib"
+    local with_ssl
+    with_ssl="$(command -v openssl)/../.."
+    export_kerl_configuration_option "--with-ssl=${with_ssl}"
```

Comment:
I wanna start simple (and it's not often I find other libs being statically linked, e.g. like `zlib` - probably they've been less prone to issues, over the years) and then maybe we can build bigger, but once we do, yeah, we have to start thinking about versioning, which is a "brave new world" in itself.